ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-vrf-lite.md

```diff
@@ -188,6 +190,8 @@ router bgp 65001
       neighbor OBS_WAN activate
       neighbor SEDI route-map RM-BGP-EXPORT-DEFAULT-BLUE-C1 out
       neighbor SEDI activate
+      neighbor SEDI-shut route-map RM-BGP-EXPORT-DEFAULT-BLUE-C1 out
+      neighbor SEDI-shut activate
```

Comment:
This is still confusing to me. Because I was expecting a ```shutdown``` command under router bgp like below:

<img width="252" alt="image" src="https://user-images.githubusercontent.com/88734745/152829643-f9d6c005-7182-4978-bdcd-06decb882498.png">